assets/template-collection.css

```diff
@@ -576,13 +582,21 @@ details[open] .mobile-facets__open {
   width: 4.4rem;
   height: 4.4rem;
   z-index: 101;
+  opacity: 0;
+  transition: opacity var(--duration-short) ease;
 }
 
-details[open] .mobile-facets__close--no-js {
+.mobile-facets__close svg {
+  margin-left: 1rem;
```

Comment:
Is this used? I couldn't find it because it's been overwritten by `margin: 0`

![image](https://user-images.githubusercontent.com/658169/125666402-2fd404bc-8236-47f3-b1bf-9c7d1ac71435.png)

Reply:
You're right - removing.